agent/engine/docker_task_engine.go

```diff
@@ -86,15 +86,19 @@ const (
 	logDriverTypeFluentd        = "fluentd"
 	logDriverTag                = "tag"
 	logDriverFluentdAddress     = "fluentd-address"
-	dataLogDriverPath           = "/data/firelens/"
+	dataLogDriverPathFirelensV1 = "/data/firelens/"
+	dataLogDriverPathFirelensV2 = "/data/telemetry/"
 	logDriverAsyncConnect       = "fluentd-async-connect"
 	logDriverSubSecondPrecision = "fluentd-sub-second-precision"
 	logDriverBufferLimit        = "fluentd-buffer-limit"
 	dataLogDriverSocketPath     = "/socket/fluent.sock"
 	socketPathPrefix            = "unix://"
 
-	// fluentTagDockerFormat is the format for the log tag, which is "containerName-firelens-taskID"
-	fluentTagDockerFormat = "%s-firelens-%s"
+	// fluentTagDockerFormat is the format for the firelens v1 log tag, which is "containerName-firelens-taskID"
+	fluentTagDockerFirelensV1Format = "%s-firelens-%s"
+
+	// fluentTagDockerFormat is the format for the firelens v2 log tag, which is "containerName-telemetry-taskID"
+	fluentTagDockerFirelensV2Format = "%s-telemetry-%s"
```

Comment:
per my original design doc, the delimiter should be dots and the format was just supposed to be `{task ID}.{container name}`

Adding telemetry in the middle is fine but unnecessary I think. Using dots as a delimiter is desired because its a standard practice in the fluent community and also customers of FireLens V1 have complained about the dashes

Reply:
Updated:)